packages/wonder-blocks-search-field/src/components/__docs__/search-field.stories.js

```diff
@@ -0,0 +1,117 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+import {action} from "@storybook/addon-actions";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+
+import type {StoryComponentType} from "@storybook/react";
+
+import SearchField from "../search-field.js";
+
+export default {
+    component: SearchField,
+    title: "Search Field/SearchField",
+    args: {id: "test-id", disabled: false, placeholder: "Placeholder"},
+};
+
+export const Default: StoryComponentType = (args) => {
+    const [value, setValue] = React.useState("");
+
+    const handleChange = (newValue: string) => {
+        setValue(newValue);
+    };
+
+    const handleKeyDown = (event: SyntheticKeyboardEvent<HTMLInputElement>) => {
+        if (event.key === "Enter") {
+            event.currentTarget.blur();
+        }
+    };
+
+    return (
+        <SearchField
+            value={value}
+            onChange={handleChange}
+            onKeyDown={(e) => {
+                action("onKeyDown")(e);
+                handleKeyDown(e);
+            }}
+            {...args}
+        />
+    );
+};
+
+export const Light: StoryComponentType = () => {
+    const [value, setValue] = React.useState("");
+
+    const handleChange = (newValue: string) => {
+        setValue(newValue);
+    };
+
+    const handleKeyDown = (event: SyntheticKeyboardEvent<HTMLInputElement>) => {
+        if (event.key === "Enter") {
+            event.currentTarget.blur();
+        }
+    };
+
+    return (
+        <View style={styles.darkBackground}>
+            <SearchField
+                id="sf-1"
+                value={value}
+                placeholder="Placeholder"
+                onChange={handleChange}
+                onKeyDown={handleKeyDown}
+                light={true}
+            />
+        </View>
+    );
+};
+
+Light.parameters = {
+    docs: {
+        storyDescription:
+            "SearchField takes a `light` prop, which gives it an extra white ring on focus to make it visible against a dark background.",
+    },
+};
+
+export const Disabled: StoryComponentType = () => {
+    const [value, setValue] = React.useState("");
+
+    const handleChange = (newValue: string) => {
+        setValue(newValue);
+    };
+
+    const handleKeyDown = (event: SyntheticKeyboardEvent<HTMLInputElement>) => {
+        if (event.key === "Enter") {
+            event.currentTarget.blur();
+        }
+    };
```

Comment:
Will this ever get called if the component is disabled?

Reply:
Great question! It should not get called if the component is disabled.

This also reminded me that we don't currently treat `TextField`/`SearchField` the same as `Clickable` when disabled, but we should. I'm gonna create a ticket for it.